core/test/Test/Pos/Core/Arbitrary/Unsafe.hs

```diff
@@ -32,9 +33,16 @@ instance ArbitraryUnsafe Address where
                 AddrAttributes
                 { aaPkDerivationPath = Nothing
                 , aaStakeDistribution = BootstrapEraDistr
+                , aaNetworkMagic = fixedNM
                 }
         let addrType = ATPubKey
         return Address {..}
 
 instance HasProtocolConstants => ArbitraryUnsafe SlotId where
     arbitraryUnsafe = SlotId <$> arbitraryUnsafe <*> arbitraryUnsafe
+
+instance ArbitraryUnsafe NetworkMagic
```

Comment:
This will also have negative ints in the field due to generic deriving.

Reply:
See https://github.com/input-output-hk/cardano-sl/pull/3558#discussion_r216471232 - I think this is OK.